src/main/java/io/cdap/plugin/marketo/common/api/entities/DateRange.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.marketo.common.api.entities;
+
+/**
+ * Represents date range.
+ */
+public class DateRange {
```

Comment:
Maybe POJO classes should have equals/toString implemented. What do you think? Your choice.

Reply:
They are not used in any collections and functions that can require this entities to be compared or printed. I will leave it as is.